Makefile

```diff
@@ -192,7 +187,7 @@ $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_A
 	./scripts/build/electron-create-resources-app.sh -s . -o $@ \
 		-v $(APPLICATION_VERSION) \
 		-f "$(APPLICATION_FILES)"
-	$(foreach prerequisite,$^,$(call execute-command,cp -rf $(prerequisite) $@))
+	$(foreach prerequisite,$^,$(call execute-command,cp -RLrf $(prerequisite) $@))
```

Comment:
Hmmm, how annoying! I found http://www.unix.com/man-page/FreeBSD/1/cp/ which says "In FreeBSD, `-r` is a synonym for `-RL` and works the same unless modified by other flags" and https://en.wikipedia.org/wiki/Cp_(Unix) which says "Using `cp -R /home/nick/clients/ /home/nick/customers` on a GNU system it behaves as expected; however, on a BSD system, it copies all the _contents_ of the "clients" directory, instead of the directory _clients_ itself. The same happens in both GNU and BSD systems if the path of the source directory ends in `.` or `..` (with or without trailing slash)."